app/templates/events/view.hbs

```diff
@@ -1,6 +1,59 @@
 {{#unless (includes session.currentRouteName 'events.view.edit')}}
-  {{!-- Tabs for the event view page will come here --}}
-  {{outlet}}
+  <div class="ui grid stackable">
+    <div class="row">
+      <div class="four wide column">
+        <h2 class="ui header">Fossasia Test</h2>
+      </div>
+      <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
+        {{#if device.isMobile}}
+          <button class="ui button"><i class="unhide icon"></i></button>
+          <div class="ui icon buttons">
+            <button class="ui button "><i class="check icon"></i></button>
+            <button class="ui button "><i class="copy icon"></i></button>
+          </div>
+          <button class="ui red button icon"><i class="trash icon"></i></button>
+        {{else}}
+            <button class="ui button labeled icon small">
+              <i class="unhide icon"></i>
+              {{t 'Preview'}}
+            </button>
+            <div class="ui small labeled icon buttons">
+              <button class="ui button ">
+                <i class="check icon"></i>
+                {{t 'Publish'}}
+              </button>
+              <button class="ui button ">
+                <i class="copy icon"></i>
+                {{t 'Copy'}}
+              </button>
+            </div>
+            <button class="ui red button labeled icon small">
+              <i class="trash icon"></i>
+              {{t 'Delete'}}
+            </button>
+        {{/if}}
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="sixteen wide column">
+      <div class="ui fluid pointing secondary menu">
```

Comment:
Please handle the ui for smaller screens. 

![screenshot from 2017-05-27 11-03-07](https://cloud.githubusercontent.com/assets/12807846/26518354/8e41d396-42cc-11e7-94cc-38fed705f78d.png)